fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/observer/JcrEvent.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.kernel.modeshape.observer;
+
+import static java.util.UUID.randomUUID;
+
+import java.util.Map;
+
+import javax.jcr.observation.Event;
+
+/**
+ * @author acoburn
+ */
+public class JcrEvent implements Event {
```

Comment:
Do you want to just cash out `FedoraEventImpl` in exchange for this class and a factory? Or putting a `static` `from(Event e)` factory method on this class that just copies the data?